src/test/java/org/dice_research/opal/civet/metrics/DateFormatMetricTest.java

```diff
@@ -0,0 +1,84 @@
+package org.dice_research.opal.civet.metrics;
+
+import org.apache.jena.ext.com.google.common.annotations.VisibleForTesting;
+import org.apache.jena.rdf.model.Model;
+import org.dice_research.opal.civet.TestData;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests {@link Dateformat}.
+ * 
+ * @author Aamir Mohammed
+ */
+public class DateFormatMetricTest {
+
+	@VisibleForTesting
```

Comment:
Why is @VisibleForTesting required?

Reply:
Codacy is giving some error, like not up to standards. So, I got this solution on google to make up to standards.